requirements/dev.txt

```diff
@@ -300,15 +300,13 @@ cryptography==44.0.0 \
     --hash=sha256:3c672a53c0fb4725a29c303be906d3c1fa99c32f58abe008a82705f9ee96f40b \
     --hash=sha256:404fdc66ee5f83a1388be54300ae978b2efd538018de18556dde92575e05defc \
     --hash=sha256:4ac4c9f37eba52cb6fbeaf5b59c152ea976726b865bd4cf87883a7e7006cc543 \
-    --hash=sha256:60eb32934076fa07e4316b7b2742fa52cbb190b42c2df2863dbc4230a0a9b385 \
```

Comment:
Any idea why these hashes have come out for `cryptography`? Should they be in this changeset?

Reply:
It's a re-compile of the requirements so I assumed some build changed, but I can remove them from this PR since it's unrelated.